From 806106d6e17f7910c63bed52a046ba4d2d158f7a Mon Sep 17 00:00:00 2001
From: Luca Boccassi <bluca@debian.org>
Date: Sat, 12 Jan 2019 12:28:54 +0000
Subject: [PATCH] man: ss.8: more line breaks

groff stiff complains about unbreakable lines:
  96: warning [p 2, 3.0i]: can't break line

Indent it some more.

Fixes: 7f5047524c99 ("man: ss.8: break and indent long line")

Signed-off-by: Luca Boccassi <bluca@debian.org>
Signed-off-by: Stephen Hemminger <stephen@networkplumber.org>
(cherry picked from commit cffeeb39466bdfcead182138e1f4c6881384ab16)
Signed-off-by: Phil Sutter <psutter@redhat.com>
---
 man/man8/ss.8 | 10 ++++++++--
 1 file changed, 8 insertions(+), 2 deletions(-)

diff --git a/man/man8/ss.8 b/man/man8/ss.8
index f138c91a1fcc4..553a6cf46f0e2 100644
--- a/man/man8/ss.8
+++ b/man/man8/ss.8
@@ -89,11 +89,17 @@ an uuid of the socket
 Show socket memory usage. The output format is:
 .RS
 .P
-skmem:(r<rmem_alloc>,rb<rcv_buf>,t<wmem_alloc>,tb<snd_buf>,f<fwd_alloc>,
+skmem:(r<rmem_alloc>,rb<rcv_buf>,t<wmem_alloc>,tb<snd_buf>,
 .br
 .RS
 .RS
-w<wmem_queued>,o<opt_mem>,bl<back_log>)
+f<fwd_alloc>,w<wmem_queued>,
+.RE
+.RE
+.br
+.RS
+.RS
+o<opt_mem>,bl<back_log>)
 .RE
 .RE
 .P
-- 
2.20.1

